tests/unit/array/mixins/test_getset.py

```diff
@@ -392,3 +393,171 @@ def test_zero_embeddings(da_cls, config, start_storage):
+        ('elasticsearch', {'n_dim': 3, 'distance': 'l2_norm'}),
+        ('sqlite', dict()),
+    ],
+)
+def test_getset_subindex(storage, config):
+
+    n_dim = 3
+    subindex_configs = (
+        {'@c': dict()} if storage in ['sqlite', 'memory'] else {'@c': {'n_dim': 2}}
+    )
+    da = DocumentArray(
+        storage=storage,
+        config=config,
+        subindex_configs=subindex_configs,
+    )
+
+    with da:
+        da.extend(
+            [
+                Document(
+                    id=str(i),
+                    embedding=i * np.ones(n_dim),
+                    chunks=[
+                        Document(id=str(i) + '_0', embedding=np.array([i, i])),
+                        Document(id=str(i) + '_1', embedding=np.array([i, i])),
+                    ],
+                )
+                for i in range(3)
+            ]
+        )
+    with da:
+        da[0] = Document(
+            embedding=-1 * np.ones(n_dim),
+            chunks=[
+                Document(id='c_0', embedding=np.array([-1, -1])),
+                Document(id='c_1', embedding=np.array([-2, -2])),
+            ],
+        )
+
+    with da:
+        da[1:] = [
+            Document(
+                embedding=-1 * np.ones(n_dim),
+                chunks=[
+                    Document(id='c_0' + str(i), embedding=np.array([-1, -1])),
+                    Document(id='c_1' + str(i), embedding=np.array([-2, -2])),
+                ],
+            )
+            for i in range(2)
+        ]
+
+    # test insert single doc
+    assert embeddings_eq(da[0].embedding, -1 * np.ones(n_dim))
+    assert embeddings_eq(da[0].chunks[0].embedding, [-1, -1])
+    assert embeddings_eq(da[0].chunks[1].embedding, [-2, -2])
+
+    assert embeddings_eq(da._subindices['@c']['c_0'].embedding, [-1, -1])
+    assert embeddings_eq(da._subindices['@c']['c_1'].embedding, [-2, -2])
+
+    # test insert slice of docs
+    assert embeddings_eq(da[1].embedding, -1 * np.ones(n_dim))
+    assert embeddings_eq(da[1].chunks[0].embedding, [-1, -1])
+    assert embeddings_eq(da[1].chunks[1].embedding, [-2, -2])
+
+    assert embeddings_eq(da._subindices['@c']['c_00'].embedding, [-1, -1])
+    assert embeddings_eq(da._subindices['@c']['c_10'].embedding, [-2, -2])
+
+    assert embeddings_eq(da[2].embedding, -1 * np.ones(n_dim))
+    assert embeddings_eq(da[2].chunks[0].embedding, [-1, -1])
+    assert embeddings_eq(da[2].chunks[1].embedding, [-2, -2])
+
+    assert embeddings_eq(da._subindices['@c']['c_01'].embedding, [-1, -1])
+    assert embeddings_eq(da._subindices['@c']['c_11'].embedding, [-2, -2])
+
+
+@pytest.mark.parametrize(
+    'storage, config',
+    [
+        ('memory', None),
+        # ('weaviate', {'n_dim': 3, 'distance': 'l2-squared'}),
+        ('annlite', {'n_dim': 3, 'metric': 'Euclidean'}),
+        # ('qdrant', {'n_dim': 3, 'distance': 'euclidean'}),
+        # ('elasticsearch', {'n_dim': 3, 'distance': 'l2_norm'}),
```

Comment:
May I ask about the reason for skipping these tests? Please at least add a reminder in the comments for the maintenance purpose.

Reply:
It's a mistake, I enabled them now